spec/features/account_creations_spec.rb

```diff
@@ -23,26 +23,27 @@
     end
   end
   # Over 21 validation added to the User model
-  context "Attempting to Make an Account" do
```

Comment:
The spec is no longer valid because the user cannot enter an age lower than 21 years. The range available for selection for the user is 21 <= age >= 116